nixos/modules/services/misc/dwm-status.nix

```
@@ -7,13 +7,9 @@
 let
   cfg = config.services.dwm-status;
 
-  order = lib.concatMapStringsSep "," (feature: ''"${feature}"'') cfg.order;
+  format = pkgs.formats.toml { };
 
-  configFile = pkgs.writeText "dwm-status.toml" ''
-    order = [${order}]
-
-    ${cfg.extraConfig}
-  '';
+  configFile = format.generate "dwm-status.toml" cfg.settings;
 in
 
 {
```

```suggestion
{
  imports = [
    (lib.mkRenamedOptionModule [ "services" "dwm-status" "order" ] [ "services" "dwm-status" "settings" "order" ])
    (lib.mkRemovedOptionModule [ "services" "dwm-status" "extraConfig" ] "Use services.dwm-status.settings instead.")
  ];
```

You still have to check if the `order` stays consistent, if not it should also be `mkRemovedOptionModule`.